dev-packages/node-integration-tests/suites/tracing/amqplib/scenario-message.ts

```diff
@@ -0,0 +1,19 @@
+import * as Sentry from '@sentry/node';
+import './init';
+import { connectToRabbitMQ, consumeMessageFromQueue, createQueue, sendMessageToQueue } from './utils';
+
+const queueName = 'queue1';
+
+// eslint-disable-next-line @typescript-eslint/no-floating-promises
+(async () => {
+  const { connection, channel } = await connectToRabbitMQ();
+  await createQueue(queueName, channel);
+
+  await Sentry.startSpan({ name: 'Test Server Start1' }, async () => {
```

Comment:
```suggestion
  await Sentry.startSpan({ name: 'root-span' }, async () => {
```

here this does not really represent the server start, so I'd call this something like this instead :)

l: IMHO you can also put the `consumeMEssageFromQueue`  call into this span, then you can just check for a single transaction that contains both of these spans correctly.